charts/kangal/README.md

```diff
@@ -1,30 +1,38 @@
-# Kangal chart
-
+# Kangal Chart
 [Kangal](https://github.com/hellofresh/kangal) is a tool to spin up an isolated environment in a Kubernetes cluster to run performance tests using JMeter
 
-## TL;DR;
```

Comment:
We should keep that in my opinion, all helm readmes have that section.

Reply:
Those instructions are still there, just a little more spaced